app/views/admin/publishers/show.html.slim

```diff
@@ -110,6 +110,8 @@
             = note.created_by.name
           .note-created-at
             = note.created_at.strftime("%B %d, %Y")
+          .note-created-at
```

Comment:
It's possible for a status change to appear after a note has been created. Additionally, it's possible for multiple status changes to occur between notes.

Reply:
This is just a CSS class, the actual functionality is that for every comment we'll look up the last corresponding publisher status that was created before the comment.
https://github.com/brave-intl/publishers/pull/1430/commits/59aac3dddc4207bec6b01d0710c4ab21380e63d6

Reply:
Oh I'm aware, I should have selected the line below. It's just that the logic for `note.publisher_status` reflects the most recent publisher status before the publisher note. It's preferable to see the sequence of actions taken against a publisher, e.g.

1) Publisher is created at a datetime 
2) Alice posts a comment at a datetime
3) Bob posts a comment at a datetime
4) Publisher's status is changed at a datetime
5) Charlie posts a comment at a datetime

This UX model is similar to Github in which we see all actions related to an issue, or if you used JIRA, it's similar to their UX model.

I realize it's a bit more work, but it's more familiar with how other issue tracking.